packages/frontend/src/Routes.tsx

```diff
@@ -225,6 +226,12 @@ const Routes: FC = () => {
                                         </TrackPage>
                                     </Route>
 
+                                    <Route path="/projects/:projectUuid/sqlRunnerNew">
```

Comment:
Should we start using lowercase routes? I find it easier to read. 
```suggestion
                                    <Route path="/projects/:projectUuid/sql-runner-new">
```

Reply:
sure! happy to do that